### DISCUSSION TITLE
How close the modal after submitting the form in php

### DISCUSSION THREAD

**Discussion by sujit15** [wireui/wireui#862] [Q&A]
public function store() {

$this->form->validate();
        User::create([
                        'name' => $this->form->name,
                        'email' => $this->form->email,
                        'password' => Hash::make($this->form->password),
                    ]);
                    $this->resetForm();
                    $this->notification()->success(
                        $title = 'Success!',
                        $description = 'User created successfully'
                    );
}

**Comment by hharyanto** [comment]
$this->modal()->close('YourModalName')
